src/com/palmergames/bukkit/towny/TownyMessaging.java

```diff
@@ -108,22 +98,30 @@ public static void sendMsg(String msg) {
 	 * @param msg the message being sent
 	 */
 	public static void sendMsg(Object sender, String msg) {
-		for (String line : ChatTools.color(TownySettings.getLangString("default_towny_prefix") + Colors.Green + msg)) {
-			if (sender instanceof Player) {
-				((Player) sender).sendMessage(line);
-			} else if (sender instanceof CommandSender) {
-				((CommandSender) sender).sendMessage(Colors.strip(line));
-			} else if (sender instanceof Resident) {
-				Player p = TownyAPI.getInstance().getPlayer((Resident) sender);
-				if (p == null) {
-					return;
-				}
-				p.sendMessage(Colors.strip(line));
-			}
+		// I don't fully understand the reason for the Resident check but I don't want to break something so...
+		if (sender instanceof Resident) {
+			Player p = TownyAPI.getInstance().getPlayer((Resident) sender);
+			if (p == null)
+				return;
+			p.sendMessage(PREFIX + ChatColor.GREEN + msg);
+		}
+		
+		if (sender instanceof Player) {
+			((Player) sender).sendMessage(PREFIX + ChatColor.GREEN + msg);
+		}
+
+		if (sender instanceof ConsoleCommandSender) {
+			((ConsoleCommandSender) sender).sendMessage(PREFIX + ChatColor.stripColor(msg));
+		}
+
+		if (sender == null) {
+			System.out.print("Message called with null sender");
 		}

```

Comment:
Use this is as probably the last else statement (if you use else-ifs above). Also do not use system.out, use TownyMessaging instead, and probably with `Invalid sender` instead of null sender if you chose to do the else-ifs.

Reply:
I'll change it, but when I forked the repo it already came with a system.out